src/Modules/CalcOffence.lua

```diff
@@ -59,6 +59,20 @@ end })
 local globalOutput = nil
 local globalBreakdown = nil
 
+local function IsTriggered(skillData)
```

Comment:
```suggestion
local function isTriggered(skillData)
```
Pretty much all functions start with lower cases in our codebase expect certain classes and a couple other exceptions which probably should be fixed.